js/packages/openinference-core/jest.config.js

```diff
@@ -3,4 +3,7 @@ module.exports = {
   preset: "ts-jest",
   testEnvironment: "node",
   prettierPath: null,
+  moduleNameMapper: {
+    "^@core/(.*)$": "<rootDir>/src/$1",
+  },
```

Comment:
Can we nix this alias for now?

Reply:
sure, just because the module is pretty small? i just didn't love the `../../` everywhere

Reply:
Oh I see. It's most commont I think to just make the root of a repo be @ I think. https://nextjs.org/docs/app/building-your-application/configuring/absolute-imports-and-module-aliases

Reply:
oh gotcha, yeah we could do that wasn't sure since these were in te monorepo if root made sense, i removed it for now